packages/ozone/src/db/migrations/20250618T150931000Z-update-appeal-reason-stats.ts

```diff
@@ -0,0 +1,309 @@
+import { Kysely, sql } from 'kysely'
```

Comment:
I think we just need to plan the migration path for ourselves and self-hosters.  E.g. what is the impact of this migration as written?

Also, do we need to do any perf testing of the updated materialized views or are we already pretty high confidence that queries wont be significantly affected.

What did the alternative path of migrating the underlying data look like as an alternative?